libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/TarResponseErrorMapTest.kt

```diff
@@ -0,0 +1,40 @@
+package com.mapbox.services.android.navigation.v5.navigation
+
+import io.mockk.every
+import io.mockk.mockk
+import java.util.HashMap
+import okhttp3.ResponseBody
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertTrue
+import org.junit.Test
+import retrofit2.Response
+
+class TarResponseErrorMapTest {
+
+    @Test
+    fun buildErrorMessage_402messageIsCreated() {
+        val errorCodes = HashMap<Int, String>()
+        val response = mockk<Response<ResponseBody>>()
+
+        every { response.code() } returns 402
+
+        val errorMap = TarResponseErrorMap(errorCodes)
+        val errorMessage = errorMap.buildErrorMessageWith(response)
```

Comment:
NIT new line
It's good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert